packages/react/src/patterns/sections/CardSectionSimple/__stories__/CardSectionSimple.stories.js

```diff
@@ -0,0 +1,31 @@
+import { object, select, withKnobs } from '@storybook/addon-knobs';
+import cards from '../../CardSection/__stories__//data/cards.json';
+import CardSectionSimple from '../CardSectionSimple';
+import React from 'react';
+import readme from '../README.md';
+import { storiesOf } from '@storybook/react';
+
+storiesOf('Patterns (Sections)|CardSectionSimple', module)
+  .addDecorator(withKnobs)
+  .addParameters({
+    readme: {
+      sidebar: readme,
+    },
+  })
+
+  .add('Default', () => {
+    const themes = {
+      white: '',
+      g10: 'g10',
+      g90: 'g90',
+      g100: 'g100',
+    };
+
+    return (
+      <CardSectionSimple
+        heading="Aliquam condimentum interdum"
+        cards={object('Data', cards.CardSectionSimple)}
+        theme={select('theme', themes, themes.g10)}
```

Comment:
One thought here is move the theme above the `cards` prop. That way it won't get lost at the bottom after the long amount of data.

Reply:
okay